jobs/apptypes_test.groovy

```diff
@@ -0,0 +1,51 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+
+name = "apptypes_e2e"
+
+
+  job(name) {
+    description """
+      runs a e2e-runner job nightly
```

Comment:
nit: I'd mention a bit more about how this would just run the `"bps"` test of the e2e suite and what that entails